### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by Morel-Victor**
1 - Add profile page with map of components in accordeonUsers (named accordeonBase) 
     AccordeonBase is call 1 time in page profile and the content are display with 5 components :  
     - Comment
     - Connexion
     - Information
     - Question
     - Ordered
     
2 - Add Information component

3 - Add Ordered component

### CODE REVIEW

**Review by Caroline-CD — APPROVED** [review]
OK :100: for me

**Review by Caroline-CD — APPROVED** [review]
OK :100: for me

**Review by Caroline-CD — DISMISSED** [review]
OK :100:

File: src/Components/Form/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 import { ToastContainer, toast } from 'react-toastify';
 import { useEffect, useState } from 'react';
-import dotenv from 'dotenv';
+// import dotenv from 'dotenv';
```
Comment (Hacene010) [inline src/Components/Form/index.jsx]: dead code

File: src/Components/AccordionUser/Ordered/style.jsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    }
+    @media (max-width: 768px) {
+      font-size: 20px;
+    }
+  }
+`;
+
+const Row2 = styled.div`
+  height: 55px;
+
```
Comment (Hacene010) [inline src/Components/AccordionUser/Ordered/style.jsx]: empty line

**Review by Hacene010 — DISMISSED** [review]

File: src/Components/Form/index.jsx (hunk 1 above)
Comment (Morel-Victor) [reply src/Components/Form/index.jsx]: done

File: src/Components/AccordionUser/Ordered/style.jsx (hunk 2 above)
Comment (Morel-Victor) [reply src/Components/AccordionUser/Ordered/style.jsx]: done

**Review by Caroline-CD — APPROVED** [review]
Ok :+1:  for me

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**